typings/externs.d.ts

```diff
@@ -83,6 +83,7 @@ declare global {
       gatherMode?: boolean | string;
       disableStorageReset?: boolean;
       disableDeviceEmulation?: boolean;
+      emulatedFormFactor?: 'mobile'|'desktop'|'none';
```

Comment:
makes me want to add a `'none'` to throttlingMethod that's just an alias of `'provided'` 😆

Reply:
> makes me want to add a `'none'` to throttlingMethod that's just an alias of `'provided'`

yes pls :D

Reply:
I think we need to as it would be confusing if this one has a none but throttling does not.
Maybe add an alias provided here too? 🙄